webui/react/src/utils/data.ts

```diff
@@ -119,7 +119,12 @@ export const metricNameSorter = (a: MetricName, b: MetricName): number => {
 };
 
 export const numericSorter = (a?: number, b?: number, reverseOrder = false): number => {
-  if (a != null && b != null) return reverseOrder ? b - a : a - b;
+  if (a != null && b != null) {
+    const diff = reverseOrder ? b - a : a - b;
+    if (diff < 0) return -1;
+    if (diff > 0) return 1;
+    return 0;
+  }
```

Comment:
there's probably a more elegant way of doing this but don't have the brain power to spend here :D

Reply:
this lg and is very readable

Reply:
this would be a great candidate and opportunity for a unit test.